src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Technically this shouldn't be necessary since a new process is created each time so there would be no breakpoints to disable.  Have you seen breakpoints persist across debugging sessions?